test/unit/spec/util/BootstrapSearchSpec.js

```diff
@@ -91,8 +91,10 @@ define(['jquery', 'util/bootstrapSearch', 'base/js/namespace'], ($, BootstrapSea
         it('Should have a working focus function', (done) => {
             const bsSearch = new BootstrapSearch($targetElem);
             $('body').append($targetElem);
-            $targetElem.find('input.form-control').on('focus', (event) => {
+            spyOn(Jupyter.narrative, 'disableKeyboardManager');
+            $targetElem.find('input[type="text"]').on('focus', (event) => {
                 expect(event.type).toBe('focus');
+                expect(Jupyter.narrative.disableKeyboardManager).toHaveBeenCalled();
```

Comment:
This test is super flaky -- not sure why. It frequently times out for no reason that I can see.